ci/computecpp.filter

```diff
@@ -1,3 +1,4 @@
+accessor
```

Comment:
Can we disable the test cases instead of the whole test?

Reply:
I added `#ifdef` command to disable part of code and `DISABLE_FOR_TEST_CASE` to disable some tests . This should allow us to remove whole test from the filter.